blob/blob.go

```diff
@@ -99,6 +99,10 @@ type Blob struct {
 	// the celestia-node's namespace type
 	// this is to avoid converting to and from app's type
 	namespace share.Namespace
+
+	// index represents index of the first share in the eds.
+	// before data is being published, the index set by default to -1.
```

Comment:
```suggestion
	// index represents index of the blob's first share in the eds.
	// before blob is posted and included onchain, the index set by default to -1.
```

Reply:
The second sentence might mean that the instance of the blob will have its index changed after posting, but that's not true, and only the retrieved blobs(not created by the user) will have it set.

Reply:
```suggestion
	// index represents index of the blob's first share in the eds.
	// -1 by default and only retrieved onchain blobs will have it set 
```